aiida_common_workflows/cli/launch.py

```diff
@@ -204,13 +210,17 @@ def cmd_eos(
             }
         }
 
+    if magnetization_per_site:
+        magnetization_per_site = list(magnetization_per_site)
```

Comment:
Why the cast to list?

Reply:
Because otherwise an error occurs. We have a validator here https://github.com/aiidateam/aiida-common-workflows/blob/7490b1c13147b4657f85652f3169beea222b4f16/aiida_common_workflows/workflows/eos.py#L89

Reply:
So imagine the value that `click` returns is a tuple, which is perfectly valid and even better. The input generator validator should simply be updated to accept _both_ a list and a tuple:
```
spec.input('generator_inputs.magnetization_per_site', valid_type=(list, tuple), required=False, non_db=True)

Reply:
Ok, just a confirmation before applying the changes. How about the same part in the "relax" command?
Again there I transform the  `magnetization_per_site` as a list. In fact, for `get_builder` of the generator we have:
https://github.com/aiidateam/aiida-common-workflows/blob/7490b1c13147b4657f85652f3169beea222b4f16/aiida_common_workflows/workflows/relax/generator.py#L108
Would you suggest any change here?

Reply:
Yes, that should then also be updated accordingly to `Any[List[float], Tuple[float]]`

Reply:
I think it is `Union[List[float], Tuple[float]]`, not `Any`. Correct?

Reply:
Yes, of course, you are right